test/integration/ovs/ovs_client_test.go

```diff
@@ -293,13 +293,12 @@ func testCreatePort(t *testing.T, br *ovsconfig.OVSBridge, name string, ifType s
 	require.Nilf(t, err, "Failed to create %s port: %s", ifType, err)
 
 	ofPort, err := br.GetOFPort(name)
-	require.Nilf(t, err, "Failed to get ofport for %s port: %s", ifType, err)
 	if ifType != "" {
+		require.Nilf(t, err, "Failed to get ofport for %s port: %s", ifType, err)
 		assert.Equal(t, ofPortRequest, ofPort, "ofport does not match the requested value for %s port", ifType)
 		ofPortRequest++
 	} else {
-		// -1 will be assigned to a port without a valid interface backing.
-		assert.Equal(t, int32(-1), ofPort)
+		require.NotNil(t, err, "GetOFPort should return an error for a port without a valid interface backing")
```

Comment:
nit: I realized recently that there was `require.NoError` available

Reply:
thanks, updated to `NoErrorf` and `Error`.